javascript/javascript1/week1/script.js

```diff
@@ -25,3 +22,61 @@ if (shouldShowResultInDogYears === true) {
     `Your dog will be ${humanYear} human years old in ${dogYearFuture}`
   );
 }
+
+//Housey pricey (A house price estimator)
+let wide = null;
+let deep = null;
+let high = null;
+let gardenSizeInM2 = null;
+
+function volumeInMeters(wide, deep, high) {
+  let volumeInMeters = wide * deep * high;
+  return volumeInMeters;
+}
+
+function housePrice(volumeInMeters, gardenSizeInM2) {
+  let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+  return housePrice;
+}
+
+//for Peter's house
+
+let peterHouseDimensions = {
+  wide: 8,
+  deep: 10,
+  high: 10,
+  gardenSizeInM2: 100,
+};
+
+let peterHouse = housePrice(volumeInMeters(8, 10, 10), 100);
+
+if (peterHouse <= 2500000) {
+  let difference = 2500000 - peterHouse;
+  console.log(
+    `Peter you are buying ${difference} more expensive than market price`
+  );
+} else {
+  let difference2 = peterHouse - 2500000;
+  console.log(`Peter you are buying ${difference2} cheaper than market price`);
+}
+
+//for Julia's House
+
+let juliaHouseDimensions = {
+  wide: 5,
+  deep: 11,
+  high: 8,
+  gardenSizeInM2: 70,
+};
+
+let juliaHouse = housePrice(volumeInMeters(5, 11, 8), 70);
+
+if (juliaHouse <= 1000000) {
+  let difference = 1000000 - juliaHouse;
+  console.log(
+    `Julia you are buying ${difference} more expensive than market price`
+  );
+} else {
+  let difference2 = juliaHouse - 1000000;
+  console.log(`Julia you are buying ${difference2} cheaper than market price`);
+}
```

Comment:
Very good job over all. You use `const` instead of `let` for defining variables.